src/components/Query.ts

```diff
@@ -91,6 +91,29 @@ export class Queries {
     }
   }
 
+  public static async bucketTagKeys (connection: InfluxDBConnection, bucket: string) {
+    if (connection.version === InfluxConnectionVersion.V1) {
+      return this.bucketTagKeysV1(connection, bucket)
+    } else {
+      return this.bucketTagKeysV2(connection, bucket)
+    }
+  }
+
+  public static async bucketTagKeysV1 (connection: InfluxDBConnection, bucket: string) {
+    const query = 'show tag keys'
```

Comment:
out of curiousity, I can't remember how bucket translates in v1. Is the DB itself the bucket?

Reply:
yeah